Test/linters/constructorVariableName.dfy

```diff
@@ -0,0 +1,42 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+datatype Color = Red | GreyScale(x: nat) | Black()
```

Comment:
Your tests should also show that the warning does not trigger if the same name constructor is not in scope, for example if it's in a different module that is not imported.